tempo/kfserving/protocol.py

```diff
@@ -6,6 +6,8 @@
 from tempo.serve.metadata import ModelDataArgs, ModelDetails
 from tempo.serve.protocol import Protocol
 
+_REQUEST_NUMPY_CONTENT_TYPE = {"content_type": "np"}
```

Comment:
Is this a MLServer specific parameter. The KFServing V2 could be for any compliant V2 server like Triton. Would need to check if this will be ignored by them?

Reply:
yes I think this is specific to MLServer. Doing a quick search on "content_type" in the triton repo doesnt result in anything that suggests that it is being used. We need to check then that it will be ignored by triton as you suggested. 

this bit is a little bit convoluted I admit, so we might want to think how to do it properly.

Reply:
The `parameters` field of the v2 payload should accept any arbitrary field. So unless Triton already uses this field for something else, I think that it should be alright.